README.md

```diff
@@ -22,21 +33,16 @@ STAGING_FIREBASE_TOKEN=
 STAGING_FIREBASE_EMAIL=
 
 # used if NODE_ENV === "dev"
-DEV_FIREBASE_TOKEN=
-DEV_FIREBASE_EMAIL=
-DEV_FIREBASE_ID=
-DEV_FIREBASE_DB_URL=
+DEV_FIREBASE_TOKEN= project settings/service accounts/generate new private key
+DEV_FIREBASE_EMAIL= project settings/services accounts/firebase service account 
+DEV_FIREBASE_ID= project settings/general/project ID
+DEV_FIREBASE_DB_URL= "https://`DEV_FIREBASE_ID`.firebaseapp.com"
```

Comment:
does this mean that the DEV_FIREBASE_DB_URL can be inferred from the DEV_FIREBASE_ID and is not really needed in the env file?

Reply:
oh good catch! You're right we can just infer url in dev-creds file